peerinst/templates/peerinst/collection/collection_distribute.html

```diff
@@ -0,0 +1,144 @@
+          You may also unassign a collection's assignments. However, this function will only unassign assignments that have not already been distributed.
+          {% endblocktrans %}
+        </small></p>
+      </div>
+    </div>
+  </div>
+
+    {% if not student_groups %}
+    <p class='mdc-typography--body1'>{% blocktrans %}You have no active student groups that may receive this collection. {% endblocktrans %}</p>
+    {% endif %}
+    <ul class="mdc-list mdc-list--two-line mdc-list--non-interactive">
+      {% for g in student_groups %}
+      <li role="separator" class="mdc-list-divider"></li>
+      <li class="mdc-list-item pointer group-link">
+        <span class="mdc-list-item__graphic mdc-theme--primary">
+          <i class="clickable mdc-theme--primary material-icons md-48" id="{{ g.hash }}">group</i>
+        </span>
+        <span class="clickable mdc-list-item__text mdc-theme--secondary bold">
+          <a id="group-title" class="mdc-theme--secondary" href="{% url 'group-details' g.hash %}">
+            {% if not g.title %} {{ g.name|bleach_html|safe }} {% else %} {{ g.title|bleach_html|safe }} {% endif %}
+          </a>
+          <span class="mdc-list-item__secondary-text">
+            {% trans "Created: " %} {{ g.creation_date }}
+          </span>
+          <span class="mdc-list-item__secondary-text">
+            {% trans "Students: " %} {{ g.student_set.count  }}
+          </span>
+        </span>
+        <span class="mdc-list-item__meta">
+          {% if group_data|get_value:g.pk is False %}
+          <button class="mdc-button mdc-button--raised collection-toggle-assign removed"
+                  id_pk="{{ g.pk }}"
+                  id_ppk="{{ object.pk }}"
+                  id="{{ g.pk }}"
+                  name="{{ g.title }}">
+                  {% trans 'Assign' %}
+          </button>
+          {% else %}
+          <button class="mdc-button mdc-button--raised collection-toggle-assign added"
+                  id_pk="{{ g.pk }}"
+                  id_ppk="{{ object.pk }}"
+                  id="{{ g.pk }}"
+                  name="{{ g.title }}">
+                  {% trans 'Unassign' %}
+          </button>
+          {% endif %}
+        </span>
+      </li>
+      {% endfor %}
+      <div class="mdc-snackbar mdc-snackbar--align-start"
+           aria-live="assertive"
+           aria-atomic="true"
+           aria-hidden="true">
+        <div class="mdc-snackbar__text bold"></div>
+        <div class="mdc-snackbar__action-wrapper" hidden>
+    <button type="button" class="mdc-snackbar__action-button"></button>
+  </div>
+      </div>
+  </section>
+</main>
+{% endblock %}
+{% block scripts %}
+{{ block.super }}
+{% compress js %}
+<script src="{% static 'peerinst/js/collection.min.js' %}"></script>
+{% endcompress %}
+<script nonce="{{ request.csp_nonce }}">
+window.addEventListener("load", function() {
+  collection.initCollectionDistribute(
```

Comment:
For new code, try organizing the parameters in categories when there are this many.